components/constellation/constellation.rs

```diff
@@ -755,6 +755,10 @@ where
         }
         self.handle_shutdown();
 
+        if !opts::get().multiprocess {
+            shutdown_style_threads();
+        }
```

Comment:
I think we need to start being careful about this kind of thing if we ever want a process to have more than a single `Servo` instance in it. Not sure what we can do about this now, but we should at least be aware that we are moving further from that goal.